module.txt

```diff
@@ -1,6 +1,6 @@
 {
     "id": "ModuleTestingEnvironment",
-    "version": "0.2.1",
```

Comment:
Deprecation of the module's main feature (so far) should be more than a patch bump 😅  as we are in 0.x land here I'd say that going to 0.3.0 should be fine.